css/info-section.css

```diff
@@ -12,6 +12,7 @@
 
 .info-section h1 {
     font-size: var(--font-size-large);
+    color: inherit;
```

Comment:
Not sure if inheriting the color is needed -- it might be an indication that you have selector specificity issues.

Reply:
All titles in the application has .title class. But in this case <h1> has to be the same color as parent .info-section

Reply:
Maybe the `.title` class is not useful to use in an `.info-section`.